src/main/java/baritone/pathing/movement/CalculationContext.java

```diff
@@ -136,7 +136,9 @@ public Block getBlock(int x, int y, int z) {
     }
 
     public double costOfPlacingAt(int x, int y, int z, IBlockState current) {
-        if (!hasThrowaway) { // only true if allowPlace is true, see constructor
+        if (!hasThrowaway || baritone.getPathingBehavior().isCloseToHome()) { /** only true if allowPlace is true, see constructor
+                                                                                * now also checks if you're not too close too home
+                                                                                */
```

Comment:
I think multi-line comments are better done above the line they belong to

Reply:
Changed in latest commit